src/report.rs

```diff
@@ -197,7 +197,7 @@ impl DefaultStringReporter {
     fn build_recursive<P: Package, VS: VersionSet>(&mut self, derived: &Derived<P, VS>) {
         self.build_recursive_helper(derived);
         if let Some(id) = derived.shared_id {
-            if self.shared_with_ref.get(&id) == None {
```

Comment:
this one looks fishy? `== None` -> `.is_some()`

Reply:
oops. fixed.